controllers/chefs/createChef/index.js

```diff
@@ -0,0 +1,16 @@
+import Chef from '../../../models/chef/Chef.model.js';
+
+export const createChef = async (req, res) => {
+  const userId = '65520e1b49c89850ff1256ea';
+
+  // const newChef = new Chef({ ...req.body, userId: req.user.id });
+  const newChefData = {
+    ...req.body,
+    userId: userId,
+  };
+
+  const newChef = new Chef(newChefData);
```

Comment:
I believe that before adding a new chef, we need to check that the user does not yet have the role of chef.

Reply:
Thanks, I've fixed that

Reply:
Sure, you are right, I'll fix that